src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -575,12 +555,14 @@ export const getInitialSwedishHouseValues = (
     yearOfConstruction: details.yearOfConstruction,
     isSubleted: details.isSubleted,
     extraBuildings: details.extraBuildings
-      .filter((b) => !!b.__typename)
-      .map((b) => ({
-        type: extraBuildingTypes[b.__typename!],
-        area: b.area,
-        hasWaterConnected: b.hasWaterConnected,
-      })),
+      .filter((building) => Boolean(building.__typename))
+      .map(({ type, area, hasWaterConnected }) => {
+        return {
+          type,
+          area,
+          hasWaterConnected,
+        }
+      }),
```

Comment:
Do we even need to do any mapping here?

Reply:
No you're probably right that we don't need that anymore.

Reply:
OK so this could be removed 😀